src/components/Tooltip/Tooltip.js

```diff
@@ -9,7 +9,8 @@ export default class Tooltip extends Component {
     open: PropTypes.bool,
     children: PropTypes.node,
     className: PropTypes.string,
-    direction: PropTypes.oneOf(['bottom', 'top']),
+    direction: PropTypes.oneOf(['bottom', 'top', 'left', 'right']),
+    menuOffset: PropTypes.obj,
```

Comment:
This should be `menuOffset: PropTypes.object,` otherwise I get a warning in my console:
```
Warning: Failed prop type: Tooltip: prop type `menuOffset` is invalid; it must be a function, usually from React.PropTypes.
```